app/views/phone/identity.xml

```diff
@@ -1,5 +1,7 @@
 <Alloy>
     <Window id="identity" title="Identity">
-        <Button id="authenticate" onClick="validate"></Button>
+        <Button id="authenticate" platform="ios" onClick="validate" />
```

Comment:
Rather than have a button and event handler per platform, could we not have a single one with the two code paths in their own blocks?

Reply:
I have optimised my code to ensure I'm reusing code where required. Unfortunately, using the (OS_IOS) block under the callback function does not seem to work well and breaks the code. I have tried different avenues to try to implement for code reusability. As the commit stands currently, I believe it's the best way to implement and works correctly.

Reply:
I have now correctly refactored the code :)